Makefile

```diff
@@ -32,3 +32,9 @@ seed:
 
 build-and-test:
 	oc create -f charts/datacenter/pipelines/extra/build-and-test-run.yaml
+
+build-and-test-iot-anomaly-detection:
+	oc create -f charts/datacenter/pipelines/extra/build-and-test-run-iot-anomaly-detection.yaml
+
+build-and-test-iot-consumer:
+	oc create -f charts/datacenter/pipelines/extra/build-and-test-run-iot-consumer.yaml
```

Comment:
I'm still not keen on any of these being part of the `Makefile`

We're actively working to kill `secret` and  `seed` is a bit of an exception that we turn a blind eye to because it allows us to deploy a functional solution.  If people need to learn something, it should be the RH products and not our Makefile (or framework) - at least not more than absolutely necessary.

Reply:
Well, I'm entirely happy with just pointing people to the `oc create` things the makefile is running.  I think it's slightly more "discoverable" in the makefile, but the documentation is going to be "run this command", either a make command or the `oc create` command the makefile would be running.  If you like we can drop the makefile targets, and just refer them to the oc create commands.

Reply:
as discussed we leave these in place for the lazy but our docs is focussed on `tkn`